### DISCUSSION TITLE
As a beginner, I would like to know how I can compile a version of Godot that only includes the 2D functionality, as I don't need any of the 3D features for my small 2D toy game. Thank you for your assistance.

### DISCUSSION THREAD

**Discussion by 4nolove** [godotengine/godot-proposals#11062] [2D]
As a beginner, I would like to know how I can compile a version of Godot that only includes the 2D functionality, as I don't need any of the 3D features for my small 2D toy game. Thank you for your assistance.

**Comment by AThousandShips** [comment]
See [optimizing for size](https://docs.godotengine.org/en/latest/contributing/development/compiling/optimizing_for_size.html#disabling-3d)
